numpy/core/src/multiarray/calculation.c

```diff
@@ -86,6 +95,28 @@ PyArray_ArgMax(PyArrayObject *op, int axis, PyArrayObject *out)
     if (ap == NULL) {
         return NULL;
     }
+
+    // Decides the shape of the output array.
+    if (!keepdims) {
+        out_ndim = PyArray_NDIM(ap) - 1;
+        out_shape = PyArray_DIMS(ap);
+    } else {
```

Comment:
```suggestion
    }
    else {
```
(I have to admit, I am also still a bit partial to `/*` style comments, but I think I have to fix my expectations there :).